pkg/cloud/services/loadbalancer/loadbalancer.go

```diff
@@ -324,6 +330,28 @@ func (s *Service) DeleteLoadBalancerMember(openStackCluster *infrav1.OpenStackCl
 	return nil
 }
 
+// DeleteOrphanedLoadBalancers deletes all remaining load balancers and all child objects within an OpenStack project.
+// Orphaned load balancers are expected when deleting the cluster because the CCM manages additional load balancers.
+func (s *Service) DeleteOrphanedLoadBalancers(openStackCluster *infrav1.OpenStackCluster) error {
+	lbList, err := loadbalancers.List(s.loadbalancerClient, loadbalancers.ListOpts{ProjectID: s.projectID}).AllPages()
```

Comment:
are we here list all LB from this project then delete all of them?

Reply:
Not sure if I understood the question correctly, but the intention here is to list all LBs from the project and delete all of them - so basically yes would be the answer.

Reply:
I think that the decision which load balancer should be deleted is part of the ongoing discussion in the linked issue. I've just added our implementation as a starting point.